pkg/actors/config.go

```diff
@@ -61,5 +69,14 @@ func NewConfig(hostAddress, appID string, placementAddresses []string, hostedAct
 		c.DrainOngoingCallTimeout = drainCallDuration
 	}
 
+	if reentrancy.Methods == nil {
+		c.Reentrancy.Methods = []string{}
+	}
```

Comment:
Is this needed? I thought a nil slice basically behaved the same as a zero-length one. 

(would be happy to be educated if I'm wrong :-D)

Reply:
I think you're right, I just prefer an empty array over a nil so I removed the nil here to avoid checks elsewhere. However, I'm going to remove methods anyway, so this will go away.